src/geo/gmaps/gmaps-torque-layer-view.js

```diff
@@ -1,9 +1,10 @@
 // Depends on cartodb standard bundle being loaded and present in global namespace
 var torque = require('torque.js');
-var cdb = window.cdb;
-var _ = cdb._;
+var cdb = require('cdb');
+var _ = require('underscore');
```

Comment:
Same case as what we discussed offline re: the leaflet-layer-view; the cdb object and libs used are "inherited" from the cartodb.js lib and should not be required here.